modules/odc-proc-binding-operator-backed-components-by-adding-a-servicebinding-resource.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * applications/connecting_applications_to_services/odc-connecting-an-application-to-a-service-using-the-developer-perspective.adoc
+:_content-type: PROCEDURE
```

Comment:
@Dhruv-Soni11 @Dhruv-Soni11 Suggest leaving a blank line after L3. Refer to [Module file metadata](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)

Reply:
Fixed.